hack/test-end-to-end.sh

```diff
@@ -105,6 +105,9 @@ function cleanup()
 	echo
 
 	if [[ -z "${SKIP_TEARDOWN-}" ]]; then
+		echo "[INFO] Switch back to 'default' project with 'admin' user for cleanup"
```

Comment:
the project switch would fail then, and it would continue cleaning up, assuming the kubeconfig file actually existed already... doesn't bother me...